src/CryingBabies.jl

```diff
@@ -61,7 +59,7 @@ end
 
 discount(p::BabyPOMDP) = p.discount
 
-function generate_o(p::BabyPOMDP, s::Bool, rng::AbstractRNG)
+function gen(::DDNOut{:o}, p::BabyPOMDP, s::Bool, rng::AbstractRNG)
```

Comment:
This should be `DDNNode`. Did you have a chance to read this yet: https://juliapomdp.github.io/POMDPs.jl/stable/generative/#The-[gen](@ref)-function-1 let me know if we can make it clearer somehow

Reply:
Actually can't we just define `observation(::BabyPOMDP, sp::Bool)` instead of `observation(::BabyPOMDP, a::Bool, sp::Bool)`?

Reply:
Ah, I hadn't read that actually, my bad! Makes sense now.
And yeah, will remove the dependence on action in `observation`